### PR TITLE
[18.0.0-proposed] Update AEE tag

### PR DESCRIPTION
**Pull request by fao89**
closes [OSPRH-10306](https://issues.redhat.com//browse/OSPRH-10306)

### CODE REVIEW

File: apis/dataplane/v1beta1/openstackdataplanenodeset_types.go [hunk 1]
```diff
@@ -207,7 +207,7 @@ func (instance OpenStackDataPlaneNodeSet) GetAnsibleEESpec() AnsibleEESpec {
 var ContainerImageDefaults = openstackv1.ContainerImages{
 	ContainerTemplate: openstackv1.ContainerTemplate{
 		AgentImage:                    getStrPtr("quay.io/openstack-k8s-operators/openstack-baremetal-operator-agent:current-podified"),
-		AnsibleeeImage:                getStrPtr("quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest"),
+		AnsibleeeImage:                getStrPtr("quay.io/openstack-k8s-operators/openstack-ansibleee-runner:18.0.0-proposed-latest"),
```
Comment (rabi) [inline apis/dataplane/v1beta1/openstackdataplanenodeset_types.go:210]: I don't think we can change it here and that too in the main branch.  It has to be updated in `OpenStackVersion` CR when deployed with 18.0.0-proposed branch I guess.

Comment (fao89) [reply apis/dataplane/v1beta1/openstackdataplanenodeset_types.go:210]: but it updates the `OpenStackVersion`: https://logserver.rdoproject.org/01/1101/34cca5e4af767ffdd79db54ab79a415fa466e7f7/github-check/podified-multinode-edpm-deployment-crc/f80d6de/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/openstackversions.core.openstack.org/controlplane.yaml

**Comment by dprince** [comment]
The image should only get updated in the 18.0.1 bundle. A new OpenStackVersion CR will then pick it up and allow the user to "update" (choose) to use the new targetVersion with this image. There should be no need to patch an image like this for a maintenance branch

**Comment by dprince** [comment]
This isn't a pattern we want to repeat for the service containers but there is a need and it is blocking us. So lets go ahead and make the exception.

**Review by dprince — APPROVED** [review]

**Pull request merged**